integrated_channels/integrated_channel/models.py

```diff
@@ -879,3 +879,48 @@ class Meta:
         on_delete=models.CASCADE,
     )
     resolved = models.BooleanField(default=False)
+
+
+class IntegratedChannelAPIRequestLogs(TimeStampedModel):
+    """
+     A model to track basic information about every API call we make from the integrated channels.
+    """
+    enterprise_customer = models.ForeignKey(
+        EnterpriseCustomer, on_delete=models.CASCADE)
+    enterprise_customer_configuration_uuid = models.UUIDField(
+        blank=False, null=False)
```

Comment:
isn't this a plain text id instead of uuid?

Reply:
oh yes,, lemme fix it! 🥇